src/relay/transforms/inline.cc

```diff
@@ -54,22 +56,36 @@ class Inliner : ExprMutator {
       : cur_node_(cur_node), call_graph_(call_graph) {}
 
   Expr VisitExpr_(const CallNode* call_node) final {
-    Expr op = call_node->op;
-    const auto* gvn = op.as<GlobalVarNode>();
+    // We can work with calls in both Relay and call_lowered form.
+    Array<Expr> args;
+    Expr op;
```

Comment:
It might be good to just insert checks below here incase the code is later refactored in a way that would make these be-unitialized.

Reply:
Introduced 'GetAnyCall' helper since this pattern was starting to multiply.